transit_odp/organisation/tests/csv/test_service_codes.py

```diff
@@ -329,251 +331,293 @@ def test_csv_output():
-    assert csv_output["row7"][13] == '"2022-01-24"'
-    assert csv_output["row7"][14] == '"2022-01-24"'
-    assert csv_output["row7"][17] == '"test7.xml"'
-    assert csv_output["row7"][18] == f'"{dataset7.id}"'
-    assert csv_output["row7"][23] == '"2023-01-24"'
-    assert csv_output["row7"][24] == '"False"'
-    assert csv_output["row7"][25] == '"2020-12-13"'
-
-    assert csv_output["row8"][0] == '"No"'
-    assert csv_output["row8"][1] == '"Published"'
-    assert csv_output["row8"][2] == '"Registered"'
-    assert csv_output["row8"][3] == '"In Scope"'
-    assert csv_output["row8"][5] == '"Not Stale"'
-    assert csv_output["row8"][6] == '"PD0000099"'
-    assert csv_output["row8"][7] == '"PD0000099:8"'
-    assert csv_output["row8"][8] == '"line1 line2"'
-    assert csv_output["row8"][9] == '"PD0000099"'
-    assert csv_output["row8"][10] == '"PD0000099:8"'
-    assert csv_output["row8"][11] == '"Line8"'
-    assert csv_output["row8"][12] == '"0"'
-    assert csv_output["row8"][13] == '"2023-02-24"'
-    assert csv_output["row8"][14] == '"2023-02-24"'
-    assert csv_output["row8"][17] == '"test8.xml"'
-    assert csv_output["row8"][18] == f'"{dataset8.id}"'
-    assert csv_output["row8"][23] == '"2024-02-24"'
-    assert csv_output["row8"][24] == '"False"'
-    assert csv_output["row8"][25] == '"2020-12-13"'
-
-    assert csv_output["row9"][0] == '"No"'
-    assert csv_output["row9"][1] == '"Published"'
-    assert csv_output["row9"][2] == '"Unregistered"'
-    assert csv_output["row9"][3] == '"In Scope"'
-    assert csv_output["row9"][4] == '"Not Seasonal"'
-    assert csv_output["row9"][5] == '"Not Stale"'
-    assert csv_output["row9"][6] == '"PD0000055"'
-    assert csv_output["row9"][7] == '"Z10"'
-    assert csv_output["row9"][8] == '"line1 line2"'
+    assert csv_output["row2"][9] == '"2023-05-12"'
+    assert csv_output["row2"][10] == '"2023-04-07"'
+    assert csv_output["row2"][11] == '""'
+    assert csv_output["row2"][12] == '""'
+    assert csv_output["row2"][13] == '""'
+    assert csv_output["row2"][14] == '""'
+    assert csv_output["row2"][15] == '""'
+    assert csv_output["row2"][16] == '""'
+    assert csv_output["row2"][17] == '""'
+    assert csv_output["row2"][18] == '""'
+    assert csv_output["row2"][19] == '""'
+    assert csv_output["row2"][20] == '""'
+    assert csv_output["row2"][21] == '""'
+    assert csv_output["row2"][22] == '"PD0000099"'
+    assert csv_output["row2"][23] == '"PD0000099:2"'
+    assert csv_output["row2"][24] == '"Line2"'
+
+    assert csv_output["row3"][0] == '"PD0000099:3"'
+    assert csv_output["row3"][1] == '"line1 line2"'
+    assert csv_output["row3"][2] == '"Yes"'
+    assert csv_output["row3"][3] == '"Published"'
+    assert csv_output["row3"][4] == '"Registered"'
+    assert csv_output["row3"][5] == '"In Scope"'
+    assert csv_output["row3"][6] == '"In Season"'
+    assert csv_output["row3"][7] == '"OTC variation not published"'
+    assert csv_output["row3"][8] == '"1"'
```

Comment:
Getting an error when running the tests

Reply:
@alvak1 What is the error you are getting? (just ran the test cases again and they seem to be passing)

Reply:
<img width="1504" alt="image" src="https://github.com/department-for-transport-BODS/bods/assets/95031756/4255296c-16df-4555-927a-6a303ba35e01">

Reply:
Resolved this @alvak1